registry.k8s.io/images/k8s-staging-jobset/images.yaml

```diff
@@ -27,3 +28,4 @@
     "sha256:ad87d7473051593ab6419c464f17641ed91607721a7dd841d5be5534b1faf3d2": ["v0.8.0"]
     "sha256:7b8a2ed7fd4c8ea3960fc16623d3328e30b7472c9166e434bea9448eb974ff32": ["v0.8.1"]
     "sha256:a25804af6def03d794c218930176c5dac12b5de06a450e38b8a15143a6fd0071": ["v0.8.2"]
+    "sha256:eeef77bf8f7548ad3727a1c1f68baf0eaf95dca52bc319ce1c69505a67244584": ["v0.9.0"]
```

Comment:
```
kehannon@kehannon-thinkpadp1gen4i:~/Work/k8s.io$ podman inspect us-central1-docker.pkg.dev/k8s-staging-images/jobset/jobset:v0.9.0
[
     {
          "Id": "365d2d4c6a14667df0e77991350e752485dfbc1bd3fe137d18c65d7d055a82b0",
          "Digest": "sha256:0aaf9c6f00f303a5587359824230b3017f25984e1b089118b6ea05501449fac1",
          "RepoTags": [
               "us-central1-docker.pkg.dev/k8s-staging-images/jobset/jobset:v0.9.0"
          ],
```

Reply:
Hmm, for me docker shows different digest, like here: https://console.cloud.google.com/artifacts/docker/k8s-staging-images/us-central1/jobset/jobset/sha256:eeef77bf8f7548ad3727a1c1f68baf0eaf95dca52bc319ce1c69505a67244584


```
$ docker inspect us-central1-docker.pkg.dev/k8s-staging-images/jobset/jobset:v0.9.0
[
    {
        "Id": "sha256:eeef77bf8f7548ad3727a1c1f68baf0eaf95dca52bc319ce1c69505a67244584",
        "RepoTags": [
            "us-central1-docker.pkg.dev/k8s-staging-images/jobset/jobset:v0.9.0"
        ],
  }
]
```